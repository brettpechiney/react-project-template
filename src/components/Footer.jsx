import React from 'react';
import styled from 'styled-components';

const Footer = () => (
  <Wrapper>
    <p>&copy; {new Date().getFullYear()} Taekni, Inc.</p>
  </Wrapper>
);

const Wrapper = styled.footer`
  background: #222;
  margin-top: 40px;
  position: fixed;
  bottom: 0;
  left: 0;
  width: 100%;
  height: 80px;
  line-height: 80px;
  font-size: 14px;
  color: #999;
  text-align: center;
`;

export default Footer;
