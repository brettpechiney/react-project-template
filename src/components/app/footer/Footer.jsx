import React from 'react';
import './footer.scss';

const Footer = () => (
  <footer styleName="app-footer">
    <p>&copy; {new Date().getFullYear()} Taekni, Inc.</p>
  </footer>
);

export default Footer;
