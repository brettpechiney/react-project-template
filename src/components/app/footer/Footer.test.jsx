import React from 'react';
import { shallow } from 'enzyme';
import Footer from './Footer';

const setup = propOverrides => {
  const props = Object.assign({}, propOverrides);

  const wrapper = shallow(<Footer {...props} />);

  return {
    props,
    wrapper,
  };
};

describe('<Footer />', () => {
  const { wrapper } = setup();

  it('renders', () => {
    expect(wrapper.exists()).toBe(true);
  });
});
