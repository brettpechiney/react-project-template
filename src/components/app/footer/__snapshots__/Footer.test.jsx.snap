// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> renders correctly 1`] = `
<footer
  className="src-components-app-footer-___footer__app-footer___mZ1YI"
>
  <p>
    Â© 
    2019
     Taekni, Inc.
  </p>
</footer>
`;
